README.md

```diff
@@ -1,73 +1,67 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Anime-World
 
-> Description the project.
+> "Anime-World" is a website show list of animes and show different facts about them.
 
+![Homepage](./application.png)
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+Design From [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29)
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- HTML
+- CSS
+- JAVASCRIPT
+- REACT
+- REDUX
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Start by cloning the repository.
 
+`git clone https://github.com/thermiee/React-Capstone.git`
 
-To get a local copy up and running follow these simple example steps.
+Navigate to the created folder.
 
-### Prerequisites
+`cd React-Capstone`
 
-### Setup
+Install all the dependencies.
 
-### Install
+`npm install`
 
-### Usage
+Run the live server
 
-### Run tests
+`npm run start`
 
-### Deployment
+Go to [Localhost](http://localhost:3000)
 
+Build the distribution folder before hosting the app
 
+`npm run build`
 
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+After the build you can host the `build` folder.
 
-👤 **Author2**
+## Authors
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Emmanuel Jolugbo**
 
+- GitHub: [Thermiee](https://github.com/thermiee)
+- LinkedIn: [Emmanuel Jolugbo](https://www.linkedin.com/in/emmanuel-jolugbo/)
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+## Acknowledgments
 
-Give a ⭐️ if you like this project!
+Thank you Microverse for the opportunity!
 
-## Acknowledgments
+## Show your support
```

Comment:
- [ ] Please ensure to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).